05/app.js

```diff
@@ -0,0 +1,152 @@
+const form = document.querySelector("form");
+const firstName = form.querySelector('input[name="firstName"]');
+const lastName = form.querySelector('input[name="lastName"]');
+const street = form.querySelector('input[name="street"]');
+const houseNumber = form.querySelector('input[name="houseNumber"]');
+const flatNumber = form.querySelector('input[name="flatNumber"]');
+const zip = form.querySelector('input[name="zip"]');
+const city = form.querySelector('input[name="city"]');
+const voivodeship = form.querySelector('select[name="voivodeship"]');
+form.noValidate = true;
+const formInputs = form.elements;
+const arr = Array.from(formInputs);
+//remove button from array
+arr.pop();
+//remove apartment number from array
+arr.splice(4, 1);
+console.log(arr);
+let errors = [];
+
+form.addEventListener("submit", checkInputs);
+
+function checkInputs(e) {
+  e.preventDefault();
+  deleteErrorStyling(errors);
+  errors = [];
+  if (!isInputEmpty(firstName) && isInputText(firstName.value)) {
+    styleSuccess(firstName);
+  } else {
+    errors.push(firstName);
+  }
+  if (!isInputEmpty(lastName) && isInputText(lastName.value)) {
+    styleSuccess(lastName);
+  } else {
+    errors.push(lastName);
+  }
+  if (!isInputEmpty(street) && isStreetNameValid(street.value)) {
+    styleSuccess(street);
+  } else {
+    errors.push(street);
+  }
+  if (isInputNumber(houseNumber.value)) {
+    styleSuccess(houseNumber);
+  } else {
+    errors.push(houseNumber);
+  }
+  if (!isInputEmpty(flatNumber.value) && isInputNumber(flatNumber.value)) {
+    styleSuccess(flatNumber);
+  } else if (!isInputEmpty(flatNumber) && !isInputNumber(flatNumber.value)) {
+    errors.push(flatNumber);
+  }
+  if (isPostalCodeValid(zip.value)) {
```

Comment:
Tutaj przekazujesz z `.value`, a wew. funkcji też masz `.value` - ogólnie proponuję ustalić jedną wersję dla wszystkich funkcji sprawdzających. 
Moim zdaniem warto przekazać do funkcji tekst, bo wtedy taka funkcja jest bardziej uniwersalna.
Jednak to powoduje zmian więc teraz już bym nie zmieniał, a następnym razem o tym pamiętał ;)